sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -3013,7 +3014,23 @@ Accept wildcard characters: False
 When set to `True`, forces a sync of **Content Security Policy** entries for SharePoint framework component in the tenant application catalog.
 New entries will be added to the configuration, if not already present, based on the `cdnBasedPath` property under a solution's `.config/write-manifests.json` if present.
 The sync may take up to 24 hours to complete.
-In multi-geo environments, **Content Security Policy** entries are unique to each geo.
+In multi-geo environments, **Content Security Policy** configuration is unique to each geo.
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Applicable: SharePoint Online
+Required: False
+Position: Named
+Default value: False
```

Comment:
I think default value will be true at one point, right? So, ok to start with false but we should remember to change this to "default == true" when we start enforcing CSP

Reply:
Currently the way this is implemented is the enforcement flight takes precedence over the flag set here. We could potentially make it so after the flight rolls out the flag is set to true in a timer job but I don't think we want to allow setting it back to false afterwards.

Reply:
No, actually I think we want to give admins the ability to switch back to report only perpetually. Power Platform folks allow that for example.